ladder/pull_request_template.md

```diff
@@ -0,0 +1,24 @@
+# Moving up the [Contributor Ladder](https://github.com/cilium/community/blob/main/CONTRIBUTOR-LADDER.md)
+
+```
```

Comment:
Looking at the rendered version, this verbatim formatting is causing the links below to not be rendered correctly.